Add WebTrends ?WT.mc_id= to URL params blacklist

seanh opened the pull request:
https://trello.com/c/hSxsjvim/222-add-webtrends-to-set-of-url-params-removed-in-normalization

nickstenning commented:
Could you find the WebTrends documentation and check if that's the only relevant query param? Also, having a reference to the documentation in a comment (as with the Google Analytics params) would be very useful.

seanh commented:
Looking into it

seanh commented:
Ok, I've added a comment with a link to the relevant WebTrends docs, and I've changed the query param blacklisting to use regexes and blacklisted `WT.*`.

All of the leading `^`s in the regexes are not strictly necessary because of what `re.match()` does but I figured it was good to be explicit and thorough with them.

nickstenning commented:
I've now fiddled with this enough that I'd appreciate a second set of eyes. @finitud?

seanh commented on h/api/uri.py:
```diff
@@ -244,3 +248,8 @@ def _normalize_queryname(name):
+
+def _blacklisted_query_param(s):
+    """Return True if the given string matches any BLACKLISTED_QUERY_PARAMS."""
+    return any(re.match(patt, s) for patt in BLACKLISTED_QUERY_PARAMS)
```
Oh, nice!
